src/BODSDataExtractor/extractor.py

```diff
@@ -68,22 +68,49 @@ def __init__(self, api_key, limit=10000, nocs=None, status='published', search=N
             self.generate_timetable()
 
         # self.service_line_extract = service_line_extract
+     
+    def check_api_response(self, response):
+
+        #initialise empty message to be appended to
+        message=""
+
+
+        if response.get("results")==[]:

```

Comment:
need a corresponding else statement for this 'if'

Reply:
Added